src/masternode.h

```diff
@@ -201,9 +195,6 @@ class CMasternode : public CSignedMessage
         READWRITE(protocolVersion);
         READWRITE(activeState);
         READWRITE(lastPing);
-        READWRITE(unitTest);
-        READWRITE(allowFreeTx);
-        READWRITE(nLastDsq);
```

Comment:
how nodes that have this stored on disk will handle this change?.

Reply:
Simple. At startup there will be an error deserializing (the log will show "Error reading mncache.dat..."), and the node will ask a new masternode list from its peers.